### ISSUE TITLE
IWR1443+DCA1000EVM

### ISSUE THREAD

**Issue by salmanpolito** [EsonJohn/mmWave_script#9]
Has someone checked this code on IWR1443+DCA1000EVM. I have followed all the steps, and successfully created the RF_API.dll, DCA1000EVM_CLI_Control.exe and DCA1000EVM_CLI_Record.exe. However, when I try to execute the command "DCA1000EVM_CLI_Control.exe fpga cf.json" the FPGA doesn't responds at all, and after a couple of seconds delay I get the following error:
FPGA Configuration :
Timeout Error! System disconnected
Kindly respond to my post if you have tested this code successfully on  IWR1443+DCA1000EVM. Your guidance will saves my lot of time. Many thanks

**Comment by DagMalstaf** [comment]
Hi @salmanpolito, I am running into the same problem. Did you manage to find a fix for this problem? If so please let me know. You can always contact me at dag.malstaf@student.kuleuven.be

**Comment by salmanpolito** [comment]
> Hi @salmanpolito, I am running into the same problem. Did you manage to find a fix for this problem? If so please let me know. You can always contact me at [dag.malstaf@student.kuleuven.be](mailto:dag.malstaf@student.kuleuven.be)

Yes, actually this code is for IWR1642, and I was trying it for IWR1443. I changed the device and then it worked. for the IWR1443, you may need to make the corresponding changes in the code
